shared_model/interfaces/transaction_responses/abstract_tx_response.hpp

```diff
@@ -41,6 +41,7 @@ namespace shared_model {
       virtual std::string className() const = 0;
 
 #ifndef DISABLE_BACKWARD
+     public:
```

Comment:
Public is not required since it's usage can be avoided as shown in test. Same applies for other responses.